src/status_im/constants.cljs

```diff
@@ -429,7 +429,11 @@
 (def ^:const optimism-short-name "opt")
 (def ^:const arbitrum-short-name "arb1")
 
-(def ^:const mainnet-network-name :ethereum)
+(def ^:const mainnet-abbreviated-name "Eth.")
+(def ^:const optimism-abbreviated-name "Opt.")
+(def ^:const arbitrum-abbreviated-name "Arb1.")
```

Comment:
Maybe instead of adding these new definitions we can use the `short-name` where first letter is captialized and adding a dot at the end

Reply:
Makes sense, but wondering how this approach would work if in the future we add support for a network with a short name.

Reply:
@briansztamfater I am not sure what do you mean? (did you mean to say without a short name?)

Reply:
I think it would still work. It seems to be that the abbreviated name is just what we call the short-name with a dot at the end. Either way is fine.